python/smqtk/algorithms/descriptor_generator/caffe_descriptor.py

```diff
@@ -181,11 +181,9 @@ def _setup_network(self):
         #   - ``caffe.TEST`` indicates phase of either TRAIN or TEST
         self._log.debug("Initializing network")
         self._log.debug("Loading Caffe network from network/model configs")
-        self.network = caffe.Net(self.network_prototxt.write_temp(),
+        self.network = caffe.Net(self.network_prototxt,
```

Comment:
`self.network_prototxt` here, as well as `self.network_model` below, should be `DataElement` instances at this stage. I believe Caffe 1 wanted filepaths here, thus the previous `write_temp()` calls. Is this not the case any more? I realize that the caffe 1 generator might be getting a little stale at this point.

Reply:
It still seems to accept a string as input. I can change the documentation in Line[https://github.com/Kitware/SMQTK/blob/654f69e0db89ce7b0a2fd13b27ef503ed4523e6b/python/smqtk/algorithms/descriptor_generator/caffe_descriptor.py#L58](url) to string and the [__setstate__ function](https://github.com/Kitware/SMQTK/blob/654f69e0db89ce7b0a2fd13b27ef503ed4523e6b/python/smqtk/algorithms/descriptor_generator/caffe_descriptor.py#L150) to reflect the same. Please let me know if you think that sounds right.